### ISSUE TITLE
integration with other workflow

### ISSUE THREAD

**Issue by thibaudart** [ZHO-ZHO-ZHO/ComfyUI-InstantID#68]
hi

your repo works well but the issue is, it's not compatible with other workflow (to have other controlnet, other ipadapter and use custom sampler).
the "all in one" node works but it's not a flexible solution.
do you think you'll modify it to have a more "comfyui" implementation?

**Comment by drphero** [comment]
I agree. It works very well on its own, but I would like to be able to include it in other workflows. The implementation for A1111 can use hires fix without changing the face, for example.
